js/src/product-feed/submission-success-guide/index.scss

```diff
@@ -66,12 +66,12 @@
 
 			.components-button {
 				position: static;
-				height: 36px;
-				padding: 0 16px;
+				height: $gla-size-control-height;
+				padding: 0 $grid-unit-20;
 				box-sizing: border-box;
-				border-radius: 3px;
+				border-radius: $gla-grid-unit-05;
```

Comment:
❔ I think it's ok to keep `border-radius` using a hardcoded value here, or maybe create another variable for it since it's a bit confusing to associate a *grid unit* to `border-radius`.

Reply:
Whats the issue using grid with border rad?

Reply:
`border-radius` is a style about the border but not the layout. Just like using a \<textarea\> to display a text block, it works but is confusing.

Reply:
Ok get it. So I created a new variable `$gla-border-radius` in here just for those. 

https://github.com/woocommerce/google-listings-and-ads/pull/1578/commits/694e94646af441a03955c6718a481e3427327dce